test/simulator/moves/followme.js

```diff
@@ -30,8 +30,7 @@ describe('Follow Me', function () {
 				hitCount++;
 			}
 		});
-		battle.choose('p2', 'move 1 2, move 1 2, move 1 2');
-		battle.commitDecisions();
+		battle.makeChoices('move followme, move calmmind, move calmmind', 'move lowkick, move lowkick, move lowkick');
```

Comment:
These are targeted Low Kicks – i.e. `move lowkick 2`

Reply:
Just updated that